src/main/java/com/red5pro/canvas/DefaultFrameProvider.java

```diff
@@ -0,0 +1,28 @@
+package com.red5pro.canvas;
+
+public class DefaultFrameProvider implements FrameProvider {
+	/**
+	 * Builder for canvas api
+	 */
+	public static CanvasFactory factory;
```

Comment:
Implement FrameProvider and call the factory for a canvas instance. Configure the canvas and roll